test/test_model_exports_to_core_aten.py

```diff
@@ -3,22 +3,28 @@
 
 import torch
 import torch._export as export
+from torch._export.verifier import ATenDialectVerifier, SpecViolationError
 
 from torch.testing._internal.common_quantization import skip_if_no_torchvision
 from torch.testing._internal.common_utils import TestCase
 
 
-def _get_ops_list(m: torch.fx.GraphModule):
-    op_list = []
-    for n in m.graph.nodes:
-        if n.op == "call_function":
-            op_list.append(n.target)
-    return op_list
+class TestPT2ETModels(TestCase):
+    def _check_core_aten_ops(self, gm: torch.fx.GraphModule):
+        """
+        Check if all ops in the graph module are core aten ops.
+        """
+        verifier = ATenDialectVerifier()
+        for node in gm.graph.nodes:
+            if (
+                node.op == "call_function"
+                and node.target not in verifier.valid_builtin_funcs()
+            ):
+                with self.assertRaises(SpecViolationError):
```

Comment:
hmm I'm confused, should this be assertNotRaise?

Reply:
yeah same question. Also I dont think this verifier will sufficiently check for core aten ops, right?

Reply:
> hmm I'm confused, should this be assertNotRaise?

Yeah, it should be `assertNotRaises`.

> I dont think this verifier will sufficiently check for core aten ops, right?

Why? I'm using `ATenDialectVerifier`, by looking at it's impl, it is checking against core aten ops. https://github.com/pytorch/pytorch/blob/275e71c562f3f1e0232e2f4a22b5388e0e468864/torch/_export/verifier.py#L117-L135

> Have you tried running this by disabling @larryliu0820's decomp? I would expect the test to fail.

I would expect it to fail as well. Don't get a chance to work on this today, let me test it tomorrow.

Reply:
oh sorry I missed that. yes `ATenDialectVerifier` should do the job.